module/VuFind/src/VuFind/RecordDriver/MarcAdvancedTrait.php

```diff
@@ -951,4 +951,57 @@ public function getCleanNBN()
         }
         return false;
     }
+
+    /**
+     * Get the full title of the record in alternative scripts.
+     *
+     * @return array
+     */
+    public function getTitlesAltScript(): array
+    {
+        return $this->getMarcReader()
+            ->getLinkedFieldsSubfields('880', '245', ['a', 'b']);
+    }
+
+    /**
+     * Get the full title of the record in alternative scripts.
+     *
+     * @return array
+     */
+    public function getFullTitlesAltScript(): array
```

Comment:
This method has exactly the same comment as getTitlesAltScript() above.  Maybe the comment should read "Get the full title (including section/part information) ...." I'm also not seeing a non-AltScript version of this method anywhere. Should we add getFullTitle() for consistency? That might help move us toward simplifying general title handling in the future, too, though maybe I'm getting ahead of myself here.